packages/service/src/storybook/utils.ts

```diff
@@ -302,71 +305,158 @@ export function createTestFiles(
     }
 }
 
+export function createChromeCapabilityWithEmulation(
+    { screen:{ width, height, dpr }, name, userAgent }: EmulatedDeviceType,
+    isHeadless: boolean,
+): WebdriverIO.Capabilities {
+    return {
+        browserName: 'chrome',
+        'goog:chromeOptions': {
+            args: [
+                'disable-infobars',
+                ...(isHeadless ? ['--headless'] : []),
+            ],
+            mobileEmulation: {
+                deviceMetrics: {
+                    width,
+                    height,
+                    pixelRatio: dpr,
+                },
+                userAgent,
+            },
+        },
+        'wdio-ics:options': {
+            logName: `local-chrome-${name.replace(/\s+/g, '-')}`,
+        },
+    }
+}
+
+/**
+ * Throw an error message if the capabilities are not set up correctly
+ */
+export function capabilitiesErrorMessage(
+    browsers: string[],
+    capabilityMap: CapabilityMap,
+    devices: string[],
+    deviceDescriptors: EmulatedDeviceType[],
+    isMobileEmulation: boolean,
+) {
+    let errorMessage = 'No capabilities were added. Please ensure that '
+    const browserIssues = browsers.some((browser:string) => !(browser in capabilityMap))
+    const deviceIssues = isMobileEmulation && devices.some((deviceName:string) => !deviceDescriptors.some(device => device.name === deviceName))
+
+    if (browserIssues && deviceIssues) {
+        errorMessage += `the browsers '${browsers.join(',')}' and devices '${devices.join(',')}' are supported.`
+    } else if (browserIssues) {
+        errorMessage += `the browsers '${browsers.join(',')}' are supported.`
+    } else if (deviceIssues) {
+        errorMessage += `the devices '${devices.join(',')}' are supported.`
+    } else {
+        errorMessage += 'the specified configuration is correct.'
+    }
+
+    throw new Error(errorMessage)
+}
+
 /**
  * Create the storybook capabilities based on the specified browsers
  */
-export function createStorybookCapabilities(capabilities: Capabilities.RemoteCapabilities, log: Logger) {
+export function createStorybookCapabilities(
+    capabilities: Capabilities.RemoteCapabilities, log: Logger,
```

Comment:
It seems that this function doesn't implement any handling of multiremote or other types of capabilities, therefor we can stricten the type a little bit here.
```suggestion
    capabilities: Capabilities.Capabilities, log: Logger,
```

Reply:
tnx, will fix that

Reply:
Assuming you mean `WebdriverIO.Capabilities`